python_modules/jupedsim/tests/test_distributions.py

```diff
@@ -235,11 +235,11 @@ def test_distribution_till_full_creates_correct_points():
     distance_to_agents = 2.0
     distance_to_polygon = 1.0
     set_seed = 1337
-    samples = distributions.distribute_til_full(polygon=polygon, distance_to_agents=distance_to_agents,
+    samples = distributions.distribute_till_full(polygon=polygon, distance_to_agents=distance_to_agents,
                                                 distance_to_polygon=distance_to_polygon,
                                                 seed=set_seed, max_iterations=750)
     # as many points created as intended
-    assert len(samples) == 18
+    assert len(samples) == 16
```

Comment:
You could parametrize this test and try it with a higher number since 16 seems to be very small

Reply:
in this test only a few points are created since it is being tested if every placed point is correct.
This means for every point is determined if it is inside the polygon as well as the distance to every other point.
Having too many points will extend to runtime of the test exponentially.
However 16 is indeed not that high.
With that in Mind. How high should that number be / how long is a test allowed to last.

Reply:
There is no correct number for tests 😄 

To keep the runtime low you could write several small tests, e.g. you could parametrize this test for different types of polygons and/or distances. So the number of created points must not be that high...

Reply:
There are now three parametrized Tests testing if points are created correctly.
All of them are not taking to long. Tests should cover what needs to be tested.